### PR TITLE
Clarify usage definition

### LINKED ISSUE #219: Clarify Usage definition
Update these pages:

- [x] estimate-job-run-time
- [x] manage-cost
- [x] max-execution-time
- [x] Check other files for uses of quantum time and update definition
- [ ] Check Cloud topics for necessary updates

### Select all that apply

- [ ] typo
- [ ] code bug
- [ ] out-of-date content
- [ ] broken link
- [X] other

### Describe the fix.

We need to clarify that usage is measured in quantum time, and that quantum time is "the duration a quantum system is committed to fulfilling a user request."

### PR DESCRIPTION
**Pull request by beckykd**
closes #219

### CODE REVIEW

**Review by abbycross — COMMENTED** [review]
Looks good!

**Review by abbycross — APPROVED** [review]

**Comment by abbycross** [comment]
@jyu00 are you ok with the language changes? If so we will merge. Thanks!

File: docs/run/estimate-job-run-time.mdx [hunk 1]
```diff
@@ -54,7 +56,7 @@ Output:
 ```
 ## View the estimated job usage on IBM Quantum Platform
 
-You can view the estimated usage (in quantum seconds) in two places on IBM Quantum Platform:
+You can view the estimated usage (in seconds) in two places on IBM Quantum Platform:
```
Comment (jyu00) [inline docs/run/estimate-job-run-time.mdx:59]: It's not at all obvious that "job usage" is the same as the quantum time that was mentioned above.

File: docs/run/estimate-job-run-time.mdx [hunk 2]
```diff
@@ -6,7 +6,9 @@ description: Estimate how long a job that uses a primitive will take to run
 
 # Estimate job run time
 
-After submitting a job to the IBM Quantum channel, you can see an estimation for how long the job will take to run in quantum seconds. You can use `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
+After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run. You can use `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
```
Comment (jyu00) [inline docs/run/estimate-job-run-time.mdx:9]: Maybe it's just me, but this sentence

> You can use `job.usage_estimation`.

seems incomplete.

Comment (abbycross) [inline docs/run/estimate-job-run-time.mdx:9]:
```suggestion
After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run, by using `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
```

File: docs/run/estimate-job-run-time.mdx [hunk 3]
```diff
@@ -6,7 +6,9 @@ description: Estimate how long a job that uses a primitive will take to run
 
 # Estimate job run time
 
-After submitting a job to the IBM Quantum channel, you can see an estimation for how long the job will take to run in quantum seconds. You can use `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
+After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run, by using `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
```
Comment (abbycross) [inline docs/run/estimate-job-run-time.mdx:9]:
```suggestion
After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run by using `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
```

File: docs/run/estimate-job-run-time.mdx (hunk 1 above)
Comment (beckykd) [reply docs/run/estimate-job-run-time.mdx]: Good point - I changed it to this "You can view the estimated usage (how much quantum time the job will take to run) in two places on IBM Quantum Platform"

File: docs/run/estimate-job-run-time.mdx [hunk 4]
```diff
@@ -6,7 +6,9 @@ description: Estimate how long a job that uses a primitive will take to run
 
 # Estimate job run time
 
-After submitting a job to the IBM Quantum channel, you can see an estimation for how long the job will take to run in quantum seconds. You can use `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
+After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run by using `job.usage_estimation`.  Alternatively, you can [view this information on the IBM Quantum Platform user interface](#view-usage).
+
+Quantum time is the duration a quantum system is committed to fulfilling a user request.
```
Comment (jyu00) [inline docs/run/estimate-job-run-time.mdx:11]:
```suggestion
Quantum time is the duration, in seconds, a quantum system is committed to fulfilling a user request.
```

**Review by jyu00 — APPROVED** [review]
One small comment but otherwise LGTM!

**Pull request merged**